examples/applications/plot_face_recognition.py

```diff
@@ -99,25 +85,27 @@
 print("done in %0.3fs" % (time() - t0))
 
 
-# #############################################################################
+# %%
 # Train a SVM classification model
-
+# %%
 print("Fitting the classifier to the training set")
 t0 = time()
 param_grid = {
     "C": [1e3, 5e3, 1e4, 5e4, 1e5],
     "gamma": [0.0001, 0.0005, 0.001, 0.005, 0.01, 0.1],
 }
```

Comment:
I think that it would make sense to change the following with distributions:

```python
param_distribution = {
    "C": loguniform(1e-5, 1e5),
    "gamma": loguniform(1e-4, 1e-1),
}
```

where `loguniform` is imported as `from sklearn.utils.fixes import loguniform`.
I checked locally and it provides the same results in average.